src/zorak/cogs/general/general_suggest.py

```diff
@@ -27,12 +27,25 @@ async def suggest(self, ctx, question):
         logger.info("%s used the %s command."
                     , ctx.author.name
                     , ctx.command)
-
-        embed = discord.Embed(description=question)
-        embed.set_author(name=f"Suggestion by {ctx.author.name}")
-        msg = await ctx.send(embed=embed)
-        await msg.add_reaction("👍")
-        await msg.add_reaction("👎")
+        
+        '''
+        
+        '''
+        embed = discord.Embed(description= question)
+        embed.set_author(name= f"Suggestion by user {ctx.author.name}")
+        error_embed = discord.Embed(title= "**Oops...**", description= "Slow down, please only use /suggest in #📎suggestions!", color= discord.Color.red())
```

Comment:
This is done perfectly, however standard practice in Zorak is to put the embeds into 
src/zorak/utilities/cog_helpers/_embeds.py

Then we use the import
`from zorak.utilities.cog_helpers._embeds import embed_suggest # pylint: disable=E0401`
and pass parameters to the embed